mica/archive/aca_l0.py

```diff
@@ -739,7 +737,7 @@ def _fetch_individual_files(self, files):
         fetched_files = []
         ingest_dates = []
         # get the files, store in file archive, and record in database
-        for file in files:
+        for i, file in enumerate(files):
```

Comment:
I don't see a dependence on the new `i`.

Reply:
Good point.  I can cut that.  I was using it for extra debugging when troubleshooting.